Doc/c-api/apiabiversion.rst

```diff
@@ -58,5 +58,13 @@ See :ref:`stable` for a discussion of API and ABI stability across versions.
    Thus ``3.4.1a2`` is hexversion ``0x030401a2`` and ``3.10.0`` is
    hexversion ``0x030a00f0``.
 
+   This version is also available via the symbol :data:`Py_Version`.
+
+.. c:var:: const unsigned long Py_Version
+
+   The Python version number encoded as the single constant integer, same value 
+   as the c:macro:`PY_VERSION_HEX` macro.
+
+   .. versionadded:: 3.11
```

Comment:
But they don't always have the same value – one is retreived runtime, the other is compiled into an extension.
```suggestion

.. c:var:: const unsigned long Py_Version

   The Python version number encoded as a single constant integer
   in the same format as the c:macro:`PY_VERSION_HEX` macro.
   This contains the Python version used at run time.

   .. versionadded:: 3.11
```

Reply:
"same format as the PY_VERSION_HEX macro" sounds like a good description. So we only have to document the format once, in PY_VERSION_HEX.

Reply:
Hmm, I'm not sure I understand this. The `PY_VERSION_HEX` is a C macro, which is then not available at runtime,  and `Py_Version` is binary constant data, that is assigned the value of `PY_VERSION_HEX` at compile time. Loosely speaking, one can use `sys.hexversion` to access `PY_VERSION_HEX` from Python, and I can see why `sys.hexversion` might not be the same as `Py_Version`. But the documentation here is stating that `PY_VERSION_HEX` and `Py_Version` have the same value at the C API level, which should always be true?

Reply:
I guess that Petr is talking about a C extension built with Python 3.8 and run with Python 3.11.

Py_Version and sys.hexversion is the version of the running Python process. PY_VERSION_HEX can be different in a C extension built with an older Python version.

I understood that the documentation can be misleading in such corner case.

Reply:
Yes.
This is what the note at the top of [the document](https://docs.python.org/3.11/c-api/apiabiversion.html) is about:

> CPython exposes its version number in the following macros. Note that these correspond to the version code is **built with**, not necessarily the version used at **run time**.

Feel free to clarify it if you find better wording :)

Reply:
Ah, I see now, thanks for clarifying!